tmt/utils.py

```diff
@@ -310,6 +327,11 @@ def get_output(self) -> str:
 _CommandElement = str
 
 
+class CommandOutput(NamedTuple):
```

Comment:
If we are already touching this part, could we move this to dataclass, seems this is the only place `NamedTuple` seems?

Reply:
That might involve more changes, I recall places where this value was expanded into two variables, `stdout, _ = run_command(...)`. I'm not sure this would be possible to preserve with a data class, and fixing the places would introduce changes unrelated to the `run_command()` implementation.

It makes sense to get rid of the `namedtuple`, I'd propose another patch for doing so.

Reply:
ok, makes sense

Reply:
Agreed.